### ISSUE TITLE
Setting login header on indexLogin.html

### ISSUE THREAD

**Issue by werkstrom** [thorsten/phpMyFAQ#1600]
Would it be possible to add a title for the text on the login page? I tried adding it myself but it didnt go to well :-D
Adding this to the language file (sv)
![image](https://user-images.githubusercontent.com/13434693/67591804-8745dd00-f75e-11e9-8360-35daf7a967d0.png)

Then this to the indexLogin.html file
![image](https://user-images.githubusercontent.com/13434693/67591864-ae041380-f75e-11e9-9eb6-bd1f878650a8.png)

Rendering me this result... LOL
![image](https://user-images.githubusercontent.com/13434693/67591784-79905780-f75e-11e9-94a8-bd64e17a1822.png)

**Comment by thorsten** [comment]
You have to add the language string key here: https://github.com/thorsten/phpMyFAQ/blob/3.0/phpmyfaq/index.php#L665

**Comment by werkstrom** [comment]
Ahh... 
Added this to index.php, $tplMainPage (row 504) as it didn't work adding it on row 665.
```
'loginPageMessage' => $PMF_LANG['loginPageMessage'],
```

Added this to the bottom of the language_sv.php

```
//Experimental - 2019-10-25 Henrik Werkström
$PMF_LANG['loginPageMessage'] = 'Logga in på ';
```

And this to _en:

```
//Experimental - 2019-10-25 Henrik Werkström
$PMF_LANG['loginPageMessage'] = 'Log in to ';
```

And finally editied row 66 and 67 in indexLogin.html like this:
```
    <img class="mb-4 rounded" src="{{ baseHref }}assets/themes/{{ tplSetName }}/img/logo.png" alt="{{ header }}">
    <h1 class="h3 mb-3 font-weight-normal">{{ loginPageMessage }} {{ header }}</h1>
```

Now looks awesome :-)
![image](https://user-images.githubusercontent.com/13434693/67691548-104b5700-f99f-11e9-954e-b4d537352ae0.png)

**Comment by thorsten** [comment]
Fixed with d2d3a6bcc654fae678d34affe9de707871bdcd24
